extensions/amp-analytics/0.1/events.js

```diff
@@ -1384,15 +1411,25 @@ export class VideoEventTracker extends EventTracker {
         event.target,
         'No target specified by video session event.'
       );
-      targetReady.then((target) => {
-        if (!target.contains(el)) {
-          return;
-        }
-        const normalizedDetails = removeInternalVars(details);
-        listener(new AnalyticsEvent(target, normalizedType, normalizedDetails));
+
+      Promise.all(targetReadyPromises).then((targetReady) => {
+        targetReady.forEach((target) => {
+          if (!target[0].contains(el)) {
```

Comment:
Can we add a check (and unit test) for when/if target is an empty array?

Reply:
It should not be necessary [once we correctly use a single `getElements()` call.](https://github.com/ampproject/amphtml/pull/34841/files#r656591863)